spec/ama_layout/navigation_spec.rb

```diff
@@ -36,29 +36,32 @@
   end
 
   describe "#items" do
```

Comment:
@sineade, i think we could either move your changes over into a new branch - or run the following command: 

```bash
git show online-account-refresh:spec/ama_layout/navigation_spec.rb > spec/ama_layout/navigation_spec.rb
```